mrtt-ui/src/views/Auth/LoginForm.js

```diff
@@ -0,0 +1,108 @@
+import { Controller, useForm } from 'react-hook-form'
+import { toast } from 'react-toastify'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+import axios from 'axios'
+
+import { ButtonCancel, ButtonSubmit } from '../../styles/buttons'
+import { ButtonContainer, RowFlexEnd } from '../../styles/containers'
+import { ErrorText } from '../../styles/typography'
+import { Form, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import { FormLabel, TextField } from '@mui/material'
+import language from '../../language'
+import LoadingIndicator from '../../components/LoadingIndicator'
+import getBaseUrl from '../../library/auth/getBaseUrl'
+
+const validationSchema = yup.object({
+  email: yup.string().required('Email required'),
+  password: yup.string().required('Password required')
+})
+
+const formDefaultValues = { email: '', password: '' }
+
+const LoginForm = () => {
+  const [isLoading] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const navigate = useNavigate()
+
+  const authUrl = `${getBaseUrl(process.env.REACT_APP_API_URL)}/users/sign_in`
```

Comment:
Im thinking for future maintainability its probably better to make a new env var rather decrypting another url to glue something together. Maybe `REACT_APP_BASE_URL=https://mrtt-api-test-2.herokuapp.com/`. Then we can remove the `getBaseUrl` utility.   Explicit over implicit.

Reply:
(sorry I didnt look at this helper function before to fully understand what it was doing)

Reply:
Ok, will remove the helper function and use env var instead. Will use `REACT_APP_AUTH_URL` as it's purely related to auth.